static/app/components/events/interfaces/spans/gapSpanDetails.tsx

```diff
@@ -8,23 +8,23 @@ import {SectionHeading} from 'sentry/components/charts/styles';
 import ExternalLink from 'sentry/components/links/externalLink';
 import {FlamegraphPreview} from 'sentry/components/profiling/flamegraph/flamegraphPreview';
 import QuestionTooltip from 'sentry/components/questionTooltip';
+import {useFlamegraphTheme} from 'sentry/domains/profiling/hooks/useFlamegraphTheme';
+import {FlamegraphThemeProvider} from 'sentry/domains/profiling/providers/flamegraphThemeProvider';
+import {useProfileGroup} from 'sentry/domains/profiling/providers/profileGroupProvider';
+import {colorComponentsToRGBA} from 'sentry/domains/profiling/utils/colors/utils';
+import {CanvasView} from 'sentry/domains/profiling/utils/profiling/canvasView';
+import {Flamegraph as FlamegraphModel} from 'sentry/domains/profiling/utils/profiling/flamegraph';
+import {getProfilingDocsForPlatform} from 'sentry/domains/profiling/utils/profiling/platforms';
```

Comment:
Any specific reason for the extra nesting here with `profiling/utils/profiling`?